crates/wdl-engine/tests/tasks/task-scope-command/config.toml

```diff
@@ -0,0 +1 @@
+backends.default = { type = "local" }
```

Comment:
Sadly, this means that I cannot check the `container` field being set. This is due to constraints in our MacOS CI environment which, apparently, cannot run container-based tests for tasks. I suppose this still gets the job done of checking that it's unset, but still unsatisfying.